test/conformance/platform/fixtures.h

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: MIT
+
+#ifndef UR_CONFORMANCE_PLATFORM_FIXTURES_H_INCLUDED
+#define UR_CONFORMANCE_PLATFORM_FIXTURES_H_INCLUDED
+
+#include <uur/fixtures.h>
+namespace uur {
+namespace platform {
+
+struct urTest : ::testing::Test {
+
+  void SetUp() override {
+    ur_platform_init_flags_t platform_flags = 0;
+    ur_device_init_flags_t device_flags = 0;
+    ASSERT_SUCCESS(urInit(platform_flags, device_flags));
+  }
+
+  void TearDown() override { ASSERT_SUCCESS(urTearDown(nullptr)); }
+};
+
+struct urPlatformsTest : urTest {
+
+  void SetUp() override {
+    UUR_RETURN_ON_FATAL_FAILURE(urTest::SetUp());
+    uint32_t count;
+    ASSERT_SUCCESS(urPlatformGet(0, nullptr, &count));
+    ASSERT_NE(count, 0);
+    platforms.resize(count);
+    ASSERT_SUCCESS(urPlatformGet(count, platforms.data(), nullptr));
+  }
+
+  std::vector<ur_platform_handle_t> platforms;
+};
+
+struct urPlatformTest : urPlatformsTest {
+
+  void SetUp() override {
+    UUR_RETURN_ON_FATAL_FAILURE(urPlatformsTest::SetUp());
+    ASSERT_GE(platforms.size(), 1);
+    platform = platforms[0]; // TODO - which to choose?
```

Comment:
Isn't the platform already chosen by `platform_options.platform_name` in PlatformEnvironment? 

I see there is uur::urPlatformTest and uur::platform::urPlatformTest. What's the reason for having both of those?

Reply:
Not in this case. This test suite does not use any of the environments (Platform or Device). The reason is that the environment is a global setup run before any tests. This kind of assumes that certain parts of the API just work. These environments will be used in future test suites and are really useful in simplifying and parametrizing tests. This is fine since we have already tested those entry points in a previous test suite.